sdk/storage/azure-storage-file-share/src/test/java/com/azure/storage/file/share/FileAPITests.groovy

```diff
@@ -2578,6 +2580,21 @@ class FileAPITests extends APISpec {
         fileClient.listHandles().size() == 0
     }
 
+    @PlaybackOnly
```

Comment:
I don't love that this is `PlaybackOnly`, is there a way where we can have this be part of live testing?

Reply:
No, it is very difficult to programmatically open a file handle, and it is prohibitively expensive in terms of engineering time to implement this in our live test pipelines.